src/main/java/teammates/common/datatransfer/attributes/GenderType.java

```diff
@@ -0,0 +1,24 @@
+package teammates.common.datatransfer.attributes;
+
+public enum GenderType {
+    male {
+        public String toString() {
+            return "male";
+        }
+    },
+    female {
+        public String toString() {
+            return "female";
```

Comment:
You don't need to override toString method for every variable. If you need to convert enum variable to string you can write something like this:
`    public String toString() {
        return name().toLowerCase();
    }`